spark-bigquery-dsv2/spark-bigquery-metrics/src/main/java/com/google/cloud/spark/bigquery/v2/customMetrics/SparkBigQueryParseTimeMetric.java

```diff
@@ -15,4 +17,9 @@ public String name() {
   public String description() {
     return BIG_QUERY_PARSE_TIME_METRIC_DESCRIPTION;
   }
+
+  @Override
+  public String aggregateTaskMetrics(long[] taskMetrics) {
+    return stringValue(taskMetrics, MetricsType.TIMING_METRIC);
```

Comment:
nit: I do not see the value of having the MetricsType enum. Instead, consider having separate methods for each of the enum values.

Reply:
+1